### DISCUSSION TITLE
BLAZING 8S @grialion

### DISCUSSION THREAD

**Discussion by javalsai** [javalsai/lidm#23] [General]
@grialion U SAID U HAD WHOLE DAY FREE, @Pickzelle IS COMPLAINING

![image](https://github.com/user-attachments/assets/9b2f552b-0131-488f-87c5-91d087da4051)

**Comment by Pickzelle** [comment]
@grialion YOU LIED!!!!!!

**Comment by javalsai** [comment]
@grialion theres some abuse going on rn in vc

![b8s](https://github.com/user-attachments/assets/5a3cb3ea-e44e-4028-ad3b-4c3419b7b796)

**Reply by Pickzelle** [reply]
skill issue

**Reply by javalsai** [reply]
> skill issue

🤓

**Reply by javalsai** [reply]
> # skill issue
![](https://cdn.discordapp.com/attachments/648577178740719616/1243934611814355014/attachment-27.gif?ex=670bd9e2&is=670a8862&hm=05b0857e67572767aa54abe9321579d8e08510413cf1e617912007ef8edb96d5&)

**Comment by javalsai** [comment]
@grialion, @Pickzelle WANTS TO PLAY KNOW WHAT I MEME LATER, UR GONNA LOOSE UR BLAZING 8S CHANCE

**Reply by javalsai** [reply]
too late now

**Reply by grialion** [reply]
Sorry, my programs for the weekend have changed and I forgot to notify you earlier
